### ISSUE TITLE
Scroll by percentage of the current page values

### ISSUE THREAD

**Issue by myself379** [philc/vimium#3314]
in Vim, there is an option to jump the scroll according to the percentage of the document.

```
{count}%
```
where `count` is in number. e.g: typing `50%` in normal mode would scroll to position 50% of the page current value. The challenge is to know the current value of page length, as some page updates the more you scroll the pages.

Excerpt from Vim documentation

```
{count}%                Go to {count} percentage in the file, on the first
                        non-blank in the line |linewise|.  To compute the new 
                        line number this formula is used:
                            ({count} * number-of-lines + 99) / 100
                        See also 'startofline' option.  {not in Vi}
```

**Comment by gdh1995** [comment]
Seems like #3296 .

Currently, `{count}scrollToTop` has be able to scroll to top and go down for `(count - 1) * scrollStepSize` pixels.

**Comment by myself379** [comment]
@gdh1995 Thanks!! I think this would work fine. So if I want to go to 50% of the page, then type `50gg` ?

**Comment by gdh1995** [comment]
No, `scrollStepSize` is a fixed number whose unit is logical pixel, and it's configured on the advanced part of Vimium options page. So you may need a count of `(target-line-number * height-per-line + top-banner-height) / scrollStepSize`.

**Comment by myself379** [comment]
@gdh1995 Ah, I see what you meant. Its actually move by pixel instead of the percentage.
